runners/spark/src/main/java/org/apache/beam/runners/spark/translation/streaming/StreamingTransformTranslator.java

```diff
@@ -115,13 +121,35 @@ public void evaluate(Read.Unbounded<T> transform, EvaluationContext context) {
     };
   }
 
-  private static <T> TransformEvaluator<CreateStream.QueuedValues<T>> createFromQueue() {
-    return new TransformEvaluator<CreateStream.QueuedValues<T>>() {
+  private static <T> TransformEvaluator<CreateStream<T>> createFromQueue() {
```

Comment:
This is somewhat unrelated to code, but as I go through this PR keeping a checklist of how far I've gotten, if the translators were (tiny) separate classes, it would be easier on my memory than remembering an offset into the file :-)